include/modes/Melee20Button.hpp

```diff
@@ -9,14 +9,12 @@ class Melee20Button : public ControllerMode {
   public:
     Melee20Button(socd::SocdType socd_type);
 
-  protected:
-    virtual void UpdateDigitalOutputs(InputState &inputs, OutputState &outputs);
-    virtual void UpdateAnalogOutputs(InputState &inputs, OutputState &outputs);
-
   private:
     bool horizontal_socd;
 
     void HandleSocd(InputState &inputs);
+    void UpdateDigitalOutputs(InputState &inputs, OutputState &outputs);
+    void UpdateAnalogOutputs(InputState &inputs, OutputState &outputs);
```

Comment:
I would suggest leaving this as protected, because to be honest there could be other reasons in the future where it is more necessary to extend and override the modes